static/webui/script.js

```diff
@@ -86,7 +86,7 @@ single.on("route_load", function (event) {
 });
 
 window.itemListPromise = new Promise(resolve => {
-    const req = $.get("/custom/getItemLists");
+    const req = $.get(`/custom/getItemLists?lang=${localStorage.getItem("lang")}`);
```

Comment:
Is this not an issue when the localStorage does not have a "lang" field?

Reply:
Nope, getString by default returns strings for en